src/__tests__/Timeline.test.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
What else can we test about this component? Think about the different states that it can be in and how you might test that it is or isn't in one of those states.